easybuild/easyconfigs/s/ScaFaCoS/ScaFaCoS-1.0.1-intel-2020a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'ScaFaCoS'
+version = '1.0.1'
+
+homepage = 'http://www.scafacos.de/'
+description = """ScaFaCoS is a library of scalable fast coulomb solvers."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = [
+    '2b125f313795c81b0e87eb920082e91addf94c17444f9486d979e691aaded99b',  # scafacos-1.0.1.tar.gz
+]
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('FFTW', '3.3.8'),
+    ('GMP', '6.2.0'),
+    ('GSL', '2.6'),
+]
+
+preconfigopts = 'unset F77 &&'
+
+configopts = ' --enable-shared --enable-static --disable-doc '
+# Tell it where to find FFTW
+configopts += ' --with-fftw3=$EBROOTFFTW '
```

Comment:
You can't get it to use fftw from mkl?

Reply:
The use case I have is for LAMMPS which is already using FFTW as a dep so I thought I might as well make it consistent (and I also didn't try with MKL because of that thinking).

Reply:
Aha. Didn't know that. then lets merge this...

Reply:
Now you made me look a little deeper and I'm not sure that LAMMPS needs to use FFTW, it can be configured to use MKL instead but this isn't currently in the easyblock. On Monday I'll make a PR for the LAMMPS easyblock and if I can get that to work I will update this too.

Reply:
Ok, I created https://github.com/easybuilders/easybuild-easyblocks/pull/1997 to update the LAMMPS easyblock to use MKL (if available) and I've updated these to use toolchain envvars for FFTW

Reply:
(Monday arrived early :P )

Reply:
@ocaisa So do we need to stick to FFTW here, or can we use the FFT wrappers in MKL?

Reply:
The FFT wrappers in MKL are (mostly) fine and that is what is being used now. The problem is that one of the solvers (p2nfft) that the original developers told me works with LAMMPS requires another dependency (pfft) that looks like being a pain to incorporate (see https://github.com/mpip/pfft/issues/36). My understanding is that this library is probably not widely used and unlikely to be maintained going forward, so it's not worth the effort and delay to do this (at least right now).